plugins/toolbar/prism-toolbar.js

```diff
@@ -63,6 +63,28 @@
 		callbacks.push(map[key] = callback);
 	};
 
+	/**
+	 * Returns the callback order of the given element.
+	 *
+	 * @param {HTMLElement} element
+	 * @returns {string[] | undefined}
+	 */
+	function getOrder(element) {
+		while (element) {
+			var order = element.getAttribute('data-toolbar-order');
+			if (order != null) {
+				order = order.trim();
+				if (order.length) {
+					return order.split(/\s*,\s*/g);
+				} else {
+					return [];
+				}
+			}
+			element = element.parentElement;
+		}
+		return undefined;
```

Comment:
We don't need to explicitly return `undefined` here. Just kill this `return`.

Reply:
Ah right. JS does that.